apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -326,6 +353,48 @@ public Response getOffers(  @Parameter(description = "Type of the offer. (BUY =
         ).build();
     }
 
+    @GET
```

Comment:
We don't need api for the matcher. It will be inner function.

Reply:
removed from public api. But some debug noise is still available there, will clean everything up after putting everything altogether